packages/rest/test/acceptance/routing/routing.acceptance.ts

```diff
@@ -416,28 +417,195 @@ describe('Routing', () => {
       .withParameter({name: 'name', in: 'query', type: 'string'})
       .build();
 
-    server.route('get', '/greet', spec, MyController, 'greet');
+    restServer.route('get', '/greet', spec, MyController, 'greet');
 
-    const client = whenIMakeRequestTo(server);
+    const client = whenIMakeRequestTo(restServer);
     await client.get('/greet?name=world').expect(200, 'hello world');
   });
 
-  /* ===== HELPERS ===== */
+  it('throws when @server is used with invalid name', async () => {
+    const app = givenAnApplication();
+    const specOne = anOpenApiSpec()
+      .withOperationReturningString('get', '/greet', 'greet')
+      .withOperationReturningString('get', '/neat', 'neat')
+      .build();
+
+    @server('RustServer')
+    @api(specOne)
```

Comment:
As I said, the `specOne` can have `servers` attribute to make `@server` unnecessary.

Reply:
We support a use-case where people can avoid the `@api` decorator entirely. Forcing users to provide a spec object in order to access this capability would be poor UX.

Reply:
I'm open for debate to have fine-grained decorators like JAX-RS. What we have in LB4 so far are coarse-grained decorators.